scripts/create_perf_json.py

```diff
@@ -1028,11 +1032,12 @@ def extract_metric_json(self, ignore, events, infoname, aux, pmu_prefix, jo, iss
                     self.save_form(em['MetricName'], em['MetricGroup'], em['MetricExpr'],
                               em['BriefDescription'], None, em.get('ScaleUnit'),
                               em.get('MetricThreshold'), [], events, infoname,
```

Comment:
perhaps "from_json=True" as the meaning of constants isn't clear

Reply:
Updated